patch-package
--- a/node_modules/@types/react-router/index.d.ts
+++ b/node_modules/@types/react-router/index.d.ts
@@ -101,7 +101,8 @@ export interface match<P> {
 
 // Diff / Omit taken from https://github.com/Microsoft/TypeScript/issues/12215#issuecomment-311923766
 export type Diff<T extends string, U extends string> = ({ [P in T]: P } & { [P in U]: never } & { [x: string]: never })[T];
-export type Omit<T, K extends keyof T> = Pick<T, Diff<keyof T, K>>;
+type Omit<T, K extends keyof T> = Pick<T, ({ [P in keyof T]: P } & { [P in K]: never } & { [x: string]: never, [x: number]: never })[keyof T]>;
+
 
 export function matchPath<P>(pathname: string, props: RouteProps): match<P> | null;
 
