patch-package
--- a/node_modules/@types/react-intl/index.d.ts
+++ b/node_modules/@types/react-intl/index.d.ts
@@ -147,7 +147,7 @@ declare namespace ReactIntl {
     }
     class FormattedMessage extends React.Component<FormattedMessage.Props> { }
 
-    class FormattedHTMLMessage extends React.Component<FormattedMessage.Props> { }
+    class FormattedHTMLMessage extends React.Component<FormattedMessage.Props & { tagName: string }> { }
 
     namespace FormattedNumber {
         interface PropsBase extends Intl.NumberFormatOptions {
