diff --git a/node_modules/@types/react-native/index.d.ts b/node_modules/@types/react-native/index.d.ts
index 5a65d89..a8c0497 100644
--- a/node_modules/@types/react-native/index.d.ts
+++ b/node_modules/@types/react-native/index.d.ts
@@ -8939,14 +8939,12 @@ export namespace Animated {
 
     export type LegacyRef<C> = { getNode(): C };
 
-    export interface WithAnimatedValue<T>
-        extends ThisType<
-            T extends object
-                ? { [K in keyof T]?: WithAnimatedValue<T[K]> }
-                : T extends (infer P)[]
+    export type WithAnimatedValue<T> = StyleProp<T extends StyleProp<infer R> ? (R extends object
+                ? { [K in keyof R]?: WithAnimatedValue<R[K]> }
+                : R extends (infer P)[]
                 ? WithAnimatedValue<P>[]
-                : T | Value | AnimatedInterpolation
-        > {}
+                : R | Value | AnimatedInterpolation) : never>
+
 
     export type AnimatedProps<T> = { [key in keyof T]: WithAnimatedValue<T[key]> } &
         (T extends {
@@ -8956,7 +8954,7 @@ export namespace Animated {
             : {});
 
     export interface AnimatedComponent<T extends React.ComponentType<any>>
-        extends React.FC<ComponentProps<T> | AnimatedProps<React.ComponentPropsWithRef<T>>> {}
+        extends React.FC<Omit<ComponentProps<T>, "style"> & { style?: WithAnimatedValue<ComponentProps<T>["style"]> }> {}
 
     /**
      * Make any React component Animatable.  Used to create `Animated.View`, etc.
@@ -8975,7 +8973,7 @@ export namespace Animated {
     /**
      * FlatList and SectionList infer generic Type defined under their `data` and `section` props.
      */
-    export class FlatList<ItemT = any> extends React.Component<FlatListProps<ItemT> | AnimatedProps<FlatListProps<ItemT>>> {}
+    export class FlatList<ItemT = any> extends React.Component<Omit<FlatListProps<ItemT>, "style"> & { style?: WithAnimatedValue<ComponentProps<T>["style"]> }> {}
     export class SectionList<SectionT = any> extends React.Component<SectionListProps<SectionT> | AnimatedProps<SectionListProps<SectionT>>> {}
 }
 
